Pages/Index.cshtml.cs

```diff
@@ -13,40 +13,69 @@ namespace NewYork_CIty_School_Data_With_Crime_Rate_History.Pages
+                //               string schoolJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
                 SchoolData[] schoolData = SchoolData.FromJson(schoolJsonString);
 
-                string crimeJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/kwvk-z7i9.json");
+                string crimeJsonString = GetData("https://data.cityofnewyork.us/resource/kwvk-z7i9.json");
+                //                string crimeJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/kwvk-z7i9.json");
                 CrimeData[] crimeData = CrimeData.FromJson(crimeJsonString);
-                              
 
-                // iterate over the specimens, to find which ones like water.
-                foreach (SchoolData schoolDatum in schoolData)
+                var schoolQuery = from SchoolData in schoolData
+                                   join CrimeData in crimeData
+                                   on SchoolData.Dbn equals CrimeData.Dbn
+                                   select new
+                                   {
+                                       Dbn = SchoolData.Dbn,
+                                       Address = CrimeData.Address
+                                   };
+                foreach (var item in schoolQuery)
                 {
-                    // find the matching plant record for this specimen.
-                    foreach (CrimeData crimeDatum in crimeData)
+                    mergedData.Add(new MergedData
                     {
-                        if (schoolDatum.Dbn == crimeDatum.Dbn)
-                        {
-                            var x = new MergedData();
-                            x.Dbn = schoolDatum.Dbn;
-                            x.Address = crimeDatum.Address;
+                        Dbn = item.Dbn,
+                        Address = item.Address
+                    });
+                }
 
-                            mergedData.Add(x);
-                        }
 
-                    }
-                }
-                                                          
+                //// iterate over the specimens, to find which ones like water.
+                //foreach (SchoolData schoolDatum in schoolData)
+                //{
+                //    // find the matching plant record for this specimen.
+                //    foreach (CrimeData crimeDatum in crimeData)
+                //    {
+                //        if (schoolDatum.Dbn == crimeDatum.Dbn)
+                //        {
+                //            var x = new MergedData();
+                //            x.Dbn = schoolDatum.Dbn;
+                //            x.Address = crimeDatum.Address;
+
+                //            mergedData.Add(x);
+                //        }
+
+                //    }
+                //}
+
             }
 
+
```

Comment:
I was wondering why you're not displaying the merged data in a UI. It would help us appreciate the merged data.